babel.config.js

```diff
@@ -72,6 +72,7 @@ module.exports = function(api) {
       "@babel/plugin-transform-flow-strip-types",
       ["@babel/proposal-class-properties", { loose: true }],
       "@babel/proposal-export-namespace-from",
+      "@babel/proposal-nullish-coalescing-operator",
```

Comment:
Nullish Coalescing is Staged 3 now. 🙏

Reply:
Can we make this `loose`?

Reply:
Sure we can apply `loose`, until we code in a too fancy way.